src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizer_provider/iot_custom_authorizer_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Hmm. Any downside to storing these materials here?

Reply:
Same question here. I'm sure there are ways to use RSA key pairs in a way that isn't a security risk. But, this is the private key. The usual Alice & Bob happy path is that Alice keeps this key in her purse, only.

The `README.md` described what to do with this `iot_custom_authorizer_key.pem`, but not where it came from. Could you add a note about that?

Reply:
Thinking about the risks of this:

- It doesn't provide any access to resources other than validating the signature of the test account's custom authorizer. A malicious user could theoretically use this to authorize WSS connections to our endpoint, from whence they would be able to publish, subscribe, and receive messages on topics. The risk would be that the attacker would be able to use the test account's resources to do pub/sub. 
- Hardcoded key materials mean it's harder to rotate if we need to reprovision the whole stack

Non-risk cons:
- AWS has infrastructure that can manage this

So yeah, I should migrate this to KMS.

It should be possible to create a KMS-key for this material. We need to push the public key with the custom authorizer, and sign the token value to pass to the integ test. I'll revise & update the PR.